man/bootctl.xml

```diff
@@ -385,6 +385,14 @@
         <xi:include href="version-info.xml" xpointer="v220"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--no-random-seed</option></term>
```

Comment:
not a fan of negated options. please change to --random-seed=BOOL instead, so that you can do --random-seed=no to achieve what you want.

Reply:
(and yeah, i know there are some pre-existing switches that didn't get this right, but let's try better for new ones)